distribution/client/test/healthchecker.test.js

```diff
@@ -17,12 +17,55 @@ describe('The healthchecker module', () => {
     }
   };
 
+  describe('constructor', () => {
+    it('should use default values if none passed in', () => {
+      const healthChecker = new HealthChecker();
+      expect(healthChecker.retry).toEqual(25);
+      expect(healthChecker.delay).toEqual(3000);
+      expect(healthChecker.factor).toEqual(1.10);
```

Comment:
Slightly unsure we should test internal default values from outside, re-harcoding them, but hopefully we won't touch this too often